src/test/functional/tests/loginPage.spec.ts

```diff
@@ -10,13 +10,13 @@ test.describe("Login Page", () => {
     await loginPage.initialize();
   });
 
-  test("Test Case 1: Login As An Admin", async ({ page }) => {
+  test("Test 1: Admin Should Be Logged In", async ({ page }) => {
     await loginPage.navigateToMainPage()
     await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password)
     await expect(loginPage.userNameAfterLogin).toHaveText(loginPageAssertion.adminName);
   });
 
-  test("Test Case 2: Login As A User", async ({ page }) => {
+  test("Test 2: User Should Be Logged In", async ({ page }) => {
     await loginPage.navigateToMainPage()
     await loginPage.loginUser(normalUserTestData.userName, normalUserTestData.password)
     await expect(loginPage.userNameAfterLogin).toHaveText(loginPageAssertion.userName);
```

Comment:
Im not a fan of your approach with file for "assertions" (which is no assertions, but simple bunch of expected strings), but as far as I can see you check that Locator with "No Posts Available" as and required text, has text "No Posts Available" from assertions file?

What I mean - we use playwright's expects (among other benefits) because they throw pretty and meaningful assertion errors.
BTW. have you tried this assertion in test that should fail on it? I mean - I can find getByText("No Posts Available") even if I have a lot of posts... Are you sure that this test works as expected?